peps/pep-0749.rst

```diff
@@ -878,9 +878,9 @@ print an empty dictionary twice, because the ``__annotate__`` function
 is set only when module execution is complete. This is obviously
 unintuitive.
```

Comment:
This is perhaps not important anymore now that the PEP is accepted, but you might want to change this sentence, given that the "obviously unintuitive" behaviour is now what PEP 749 implements.

Reply:
It's not what's currently implemented.